src/app/zap-templates/zcl/data-model/chip/mode-select-cluster.xml

```diff
@@ -38,26 +69,51 @@ limitations under the License.
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <description>Attributes and commands for selecting a mode from a list of supported options.</description>
+    <globalAttribute side="either" code="0xFFFD" value="2"/>
     <!-- Base data types -->
-    <attribute side="server" code="0x0000" define="MODE_DESCRIPTION" type="CHAR_STRING"                                     writable="false" optional="false" isNullable="false" length="32">Description</attribute>
+    <attribute side="server" code="0x0000" define="MODE_DESCRIPTION" type="CHAR_STRING"                                     writable="false" optional="false" isNullable="false" length="64">Description</attribute>
     <attribute side="server" code="0x0001" define="STANDARD_NAMESPACE" type="ENUM16"                                        writable="false" optional="false"  isNullable="true">StandardNamespace</attribute>
```

Comment:
So Cam says the D in the updated spec means "remove".  We should do that.

Reply:
I removed all `STANDARD_NAMESPACE` attribute entries.